### DISCUSSION TITLE
Negligible methylation in mitochondrial sequence

### DISCUSSION THREAD

**Discussion by yaaminiv** [RobertsLab/resources#1169] [Q&A] [answered]
I created an [IGV session](https://github.com/RobertsLab/project-gigas-oa-meth/blob/master/output/05-bismark-roslin/igv_session.xml) to look at methylation in the mitochondrial sequence. This is the mitochondrial sequence I got from NCBI, not the separate sequence from the Roslin genome paper. The genome/index file I created with IGV can be found [here](https://gannet.fish.washington.edu/spartina/project-oyster-oa/data/Cg-roslin/)

![Screen Shot 2021-03-30 at 12 52 35 PM](https://user-images.githubusercontent.com/22335838/113047979-db749e00-9156-11eb-89a9-26ad5cdb8aee.png)

The mitochondrial sequence is predominantly unmethylated, but there are some methylated CpGs. Is this a sign that something when wrong in the alignment process?

**Comment by sr320** [comment]
What is the coverage threshold here?

**Reply by yaaminiv** [reply]
These are bedgraphs from `bismark`, no coverage threshold

**Reply by kubu4** [reply]
Also, make sure you're not viewing the mean values (IGV enables the mean view sometimes).

**Reply by yaaminiv** [reply]
> Also, make sure you're not viewing the mean values (IGV enables the mean view sometime

Shoot good point. Here's the same region with no windowing function:

![Screen Shot 2021-03-30 at 1 20 52 PM](https://user-images.githubusercontent.com/22335838/113051396-c26dec00-915a-11eb-87b6-c87d4a1f59c6.png)

**Reply by sr320** [reply]
I would set threshold at 5x, and look again.

**Comment by yaaminiv** [comment, marked as answer]
I used 5x bedgraphs. Definitely better, but there are still traces of methylation

Full mitochondrial sequence:

<img width="1344" alt="Screen Shot 2021-04-04 at 1 14 16 PM" src="https://user-images.githubusercontent.com/22335838/113520425-d98c4f80-9547-11eb-9cc0-612091886d0f.png">

Area 1:
<img width="1344" alt="Screen Shot 2021-04-04 at 1 14 32 PM" src="https://user-images.githubusercontent.com/22335838/113520429-e01ac700-9547-11eb-8534-eef968f96916.png">

Area 2:

<img width="1344" alt="Screen Shot 2021-04-04 at 1 14 51 PM" src="https://user-images.githubusercontent.com/22335838/113520444-f32d9700-9547-11eb-8b8b-2eedd36cb24e.png">

How should I proceed? I could remove these locations from my dataset, try increasing coverage to 10x to see if they're still there, or re-map to the Roslin mitochondrial sequence.

**Reply by yaaminiv** [reply]
Good to keep moving forward

At some point, should check bisulfite conversion efficiency with spike in/mitochondrial methylation rate
